gax-java/gax-httpjson/src/test/java/com/google/api/gax/httpjson/InstantiatingHttpJsonChannelProviderTest.java

```diff
@@ -118,8 +118,12 @@ void managedChannelUsesDefaultChannelExecutor() throws IOException {
     // By default, the channel will be wrapped with ManagedHttpJsonInterceptorChannel
     ManagedHttpJsonInterceptorChannel interceptorChannel =
         (ManagedHttpJsonInterceptorChannel) httpJsonTransportChannel.getManagedChannel();
-    ManagedHttpJsonChannel managedHttpJsonChannel = interceptorChannel.getChannel();
-    assertThat(managedHttpJsonChannel.getExecutor()).isNotNull();
+    // call getChannel() twice because interceptors are chained in layers by recursive construction
+    // inside com.google.api.gax.httpjson.InstantiatingHttpJsonChannelProvider.createChannel
+    ManagedHttpJsonInterceptorChannel managedHttpJsonChannel =
+        (ManagedHttpJsonInterceptorChannel) interceptorChannel.getChannel();
+    ManagedHttpJsonChannel channel = managedHttpJsonChannel.getChannel();
+    assertThat(channel.getExecutor()).isNotNull();
```

Comment:
Can we add a new test and leave the existing tests untouched?

Reply:
no, the original test would fail.

Reply:
Could you add a comment about changes in the exising test cases?